### PR TITLE
add support for literal multi-line strings

### LINKED ISSUE #31: Multi-line string
it seems the parser converts multiline string to simple string, e.g.
```
description: |
  Some information
```
will be saved internally as
```
description: | Some description
```
My assumption is that it should be like
```
description: |\n  Some description
```

### PR DESCRIPTION
**Pull request by lovebug356**
Added support for the literal multi-line strings as described in the yaml specification:
https://yaml.org/spec/1.2.2/#812-literal-style.

Closes #31

### CODE REVIEW

File: tests/TestParser.elm
Comment (MaybeJustJames) [inline tests/TestParser.elm:1]: Thank you very much for doing this @lovebug356 ! This is great work :)

There seem to be a bunch of formatting changes in `tests/TestParser.elm`? If you can remove these unrelated formatting changes I'm very happy to merge.

**Comment by lovebug356** [comment]
Ok, thank should be fixed now.

**Comment by MaybeJustJames** [comment]
Very nice @lovebug356 . Thank you. Sorry it took me so long to get back to this.
